### ISSUE TITLE
can't change admin password / returns 404

### ISSUE THREAD

**Issue by Zixim** [Phantomias2006/Nemesis#110]
trying to change the password from [http://192.168.11.109/admin](http://192.168.11.109/admin) results in a 404 : 
```
No webpage was found for the web address: http://192.168.11.109/setadmin
HTTP ERROR 404
```

software is version 1.0.6

**Comment by Zixim** [comment]
`/setadmin` is missing in c_webhandler.h

**Comment by sochs** [comment]
Yeah, it's a known bug, will be fixed with next release. If you want to use it now, you have to send a POST request with parameter directly to /admin instead of /setadmin.
